src/java/org/apache/cassandra/io/sstable/format/SSTableReader.java

```diff
@@ -66,17 +66,8 @@
 import org.apache.cassandra.concurrent.NamedThreadFactory;
 import org.apache.cassandra.concurrent.ScheduledExecutors;
 import org.apache.cassandra.config.DatabaseDescriptor;
-import org.apache.cassandra.db.Clustering;
-import org.apache.cassandra.db.ClusteringBoundOrBoundary;
-import org.apache.cassandra.db.ClusteringPrefix;
-import org.apache.cassandra.db.ColumnFamilyStore;
-import org.apache.cassandra.db.DataRange;
-import org.apache.cassandra.db.DecoratedKey;
-import org.apache.cassandra.db.DeletionTime;
-import org.apache.cassandra.db.PartitionPosition;
-import org.apache.cassandra.db.SerializationHeader;
-import org.apache.cassandra.db.Slices;
-import org.apache.cassandra.db.SystemKeyspace;
+import org.apache.cassandra.db.*;
```

Comment:
Here and below, just checking if this was intended or done by the IDE.

Reply:
Reverted.